enterprise/ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -71,7 +70,7 @@ public void testSslReqHandler() {
     }
 
     private static void sendSslRequest(EmbeddedChannel channel) {
-        ByteBuf buffer = releaseLater(Unpooled.buffer());
+        ByteBuf buffer = Unpooled.buffer();
```

Comment:
I think `channel.writeInbound` auto-releases it - if I manually release it, it fails because the counter gets negative